src/contracts/pods/EigenPod.sol

```diff
@@ -44,6 +44,12 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
     /// @notice Maximum "staleness" of a Beacon Chain state root against which `verifyBalanceUpdate` or `verifyWithdrawalCredental` may be proven.
     uint256 internal constant VERIFY_BALANCE_UPDATE_WINDOW_SECONDS = 4.5 hours;
 
+    //this is the genesis time of the beacon state, to help us calculate conversions between slot and timestamp
+    uint64 internal constant GENESIS_TIME = 1616508000;
```

Comment:
this needs to be a contruction parameter?

Reply:
yeah fair, fixed